src/assets/wise5/classroomMonitor/dataExport/export-events/export-events.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { AbstractExportComponent } from '../abstract-export.component';
+import { EventDataExportStrategy } from '../strategies/EventDataExportStrategy';
+
+@Component({
+  selector: 'export-events',
+  templateUrl: './export-events.component.html',
+  styleUrls: ['./export-events.component.scss']
+})
+export class ExportEventsComponent extends AbstractExportComponent {
+  public includeNames: boolean = false;
+  public includeStudentEvents: boolean = true;
+  public includeTeacherEvents: boolean = true;
```

Comment:
Do these fields need to be public?

Reply:
These need to be public due to the coupling of the strategy with the controller. Other export strategies have this problem too. We should refactor all of them later after all the exports have their own route/component.

https://github.com/WISE-Community/WISE-Client/blob/0c29088c2b2f30180b2a3d8b1c7275f961257482/src/assets/wise5/classroomMonitor/dataExport/strategies/EventDataExportStrategy.ts#L6-L17

Reply:
Ok. Sounds good.